task2.py

```diff
@@ -0,0 +1,47 @@
+import json
+from pprint import pprint
+
+
+# function to get textures of given set
+def getTextures(_set,source):
```

Comment:
Variables start with '_' are considered as unused variables. Make sure to use this practice in your code.